src/ai/backend/manager/api/gql/artifact.py

```diff
@@ -81,6 +85,89 @@
 from .artifact_registry_meta import ArtifactRegistryMeta
 
 
+@strawberry.type(
+    description=dedent_strip("""
+    Added in 25.17.0.
+
+    Result from a single malware verifier containing scan results and metadata.
+
+    Each verifier scans the artifact for potential security issues and reports
+    findings including infected file count, scan time, and any errors encountered.
+    """)
+)
+class ArtifactSingleVerifierResult:
+    success: bool = strawberry.field(description="Whether the verification completed successfully")
+    infected_count: int = strawberry.field(
+        description="Number of infected or suspicious files detected"
+    )
+    scanned_at: datetime = strawberry.field(description="Timestamp when verification started")
+    scan_time: float = strawberry.field(
+        description="Time taken to complete verification in seconds"
+    )
+    scanned_count: int = strawberry.field(description="Total number of files scanned")
+    error: Optional[str] = strawberry.field(
+        description="Fatal error message if the verifier failed to complete"
+    )
```

Comment:
Do you transmit the parsed value even though the scan result is not visible in the field?